src/core/core.platform.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// Core.Platform abstracts away browser specific APIs from the chart
+module.exports = function(Chart) {
+	var platform = Chart.corePlatform = {};
+
+	/**
+	 * @name Core.platform.events
+	 * @type Map<String, String>
+	 */
+	platform.events = {
+		mouseenter: 'mouseenter',
+		mousemove: 'mousemove',
+		mouseout: 'mouseout',
+		mousedown: 'mousedown',
+		mouseup: 'mouseup',
+		click: 'click',
+		dblclick: 'dblclick',
+		contextmenu: 'contextmenu',
+		keydown: 'keydown',
+		keypress: 'keypress',
+		keyup: 'keyup'
+	};
+
+	/**
+	 * @interface IPlatform
+	 * Allows abstracting platform dependencies away from the chart
+	 */
+	/**
+	 * Creates a chart.js event from a platform specific event
+	 * @method IPlatform#createEvent
+	 * @param e {Event} : the platform event to translate
+	 * @returns {Core.Event} chart.js event
+	 */
+	/**
+	 * @method IPlatform#acquireContext
+	 * @param item {CanvasRenderingContext2D|HTMLCanvasElement} the context or canvas to use
```

Comment:
Item type should be `Object` since on other platforms it might be anything (e.g. in QML, it's an QML Item). Also missing the doc for the return type: `@returns A context2d instance implementing the W3C Canvas 2D Context API standard`